src/ReturnBookUi.java

```diff
@@ -25,25 +50,26 @@ public void run() {
                     break;
                 }
                 case READY: {
-                    String bookStr = input("Scan Book (<enter> completes): ");
-                    if (bookStr.length() == 0) {
+                    String bookIdString = getInput("Scan Book (<enter> completes): ");
```

Comment:
Add a new line in between lines 53 and 54 to make spacing uniform throughout file.